contrib/debian/rules

```diff
@@ -18,18 +18,7 @@ endif
 
 SB_STYLE := debian
 deb_version := $(shell dpkg-parsechangelog --show-field Version)
-ifeq (yes,$(shell dpkg-vendor --derives-from Ubuntu && echo yes))
-	SB_STYLE := ubuntu
-	tar_name := fwupd_$(deb_version)_$(DEB_HOST_ARCH).tar.gz
-	CONFARGS += -Dplugin_flashrom=false
-else
-	TMPLDIR  := debian/fwupd-$(DEB_HOST_ARCH)-signed-template/usr/share/code-signing/fwupd-$(DEB_HOST_ARCH)-signed-template
-	ifneq ($(DEB_HOST_ARCH_CPU),ia64)
-		CONFARGS += -Dplugin_flashrom=true
-	else
-		CONFARGS += -Dplugin_flashrom=false
-	endif
-endif
+CONFARGS += -Dplugin_flashrom=true
```

Comment:
Here is the MIR process: https://wiki.ubuntu.com/MainInclusionProcess
You'll need to get that approved before we can pull in this specific change.

Reply:
It's done!
https://bugs.launchpad.net/ubuntu/+source/flashrom/+bug/1912371